### PR TITLE
Add endpoint for exposing current consensus node configuration

### PR DESCRIPTION
**Pull request by eranrund**
### Motivation

The addition of minting support increased the complexity of the consensus-service configuration, especially when it comes to the dynamic nature of it - the currently active mint configurations (which change over time as `MintConfigTx` transactions are submitted).
I think it will be handy to have a way of querying a node for its current configuration, in particular when it comes to debugging why minting or mint config transactions do not go through (if that ever happens, which due to the changing nature of the configuration, is a possible scenario).

### In this PR
* Adding protobuf objects for representing a select subset of consensus service configuration
* Adding a GRPC endpoint for querying the current configuration

### CODE REVIEW

**Review by remoun — APPROVED** [review]

File: consensus/api/proto/consensus_client.proto [hunk 1]
```diff
@@ -75,4 +77,7 @@ service ConsensusClientAPI {
 
     /// Propose a new MintTx.
     rpc ProposeMintTx(external.MintTx) returns (ProposeMintTxResponse);
+
+    /// Get current node configuration.
+    rpc GetNodeConfig(google.protobuf.Empty) returns (consensus_config.ConsensusNodeConfig);
```
[the remark this answers is not in the document]
Comment (eranrund) [reply consensus/api/proto/consensus_client.proto:82]: I don't think this is necessary - it's unlikely we will need to include any parameters in this Get request, and if we ever do then while being a breaking change it is very minor and easy to address.
We've been using Empty liberally in other places and have not ran into issues so far. Someone mentioned "proto fatigue" during the block streaming walkthrough, so I think adding another empty type here for potential future compatibility would just contribute to that.

With that said, I don't feel strongly about this.

[the remark this answers is not in the document]
Comment (remoun) [reply consensus/api/proto/consensus_client.proto:82]: SGTM

**Review by iamalwaysuncomfortable — APPROVED** [review]
Seems great, although is there any case where we'd want a mint config to be private?

**Comment by eranrund** [comment]
> Seems great, although is there any case where we'd want a mint config to be private?

No, quite the opposite - we want as much transparency as possible around minting (and burning).

**Pull request merged**